integration_tests/helpers/walletProcess.js

```diff
@@ -57,7 +49,7 @@ class WalletProcess {
                 fs.mkdirSync(this.baseDir + "/log", {recursive: true});
             }
 
-           let envs = createEnv("nodeid.json","127.0.0.1",this.grpcPort, this.baseNodeGrpcAddress, this.baseNodeGRPCPort,this.baseNodePort, "127.0.0.1:8080", [], this.peerSeeds)
+           let envs = createEnv(this.name,true, "nodeid.json","127.0.0.1",this.grpcPort, this.port, this.baseNodeGrpcAddress, this.baseNodeGRPCPort,this.baseNodePort, "127.0.0.1:8081", [], this.peerSeeds)
```

Comment:
Code can be simplified.

Suggest only pass non-default values as named parameters.